src/test/java/org/opensearch/knn/TestUtils.java

```diff
@@ -32,6 +30,62 @@
     public static final String BWCSUITE_ROUND = "tests.rest.bwcsuite_round";
     public static final String TEST_CLUSTER_NAME = "tests.clustername";
 
+    public static float[][] generateVectors(int numVectors, int dimensions) {
```

Comment:
I think generateVectors and generateQueryVectors function names should be refactored. By the name, it is not clear what makes them different. generateRandomVectors is descriptive in that it says how it is generating the vectors.

Reply:
updated